app/services/idv/steps/verify_wait_step_show.rb

```diff
@@ -29,8 +29,9 @@ def process_async_state(current_async_state)
       def async_state_done(current_async_state)
         add_proofing_costs(current_async_state.result)
         form_response = idv_result_to_form_response(
-          idv_result: current_async_state.result,
+          result: current_async_state.result,
           state: flow_session[:pii_from_doc][:state],
```

Comment:
as long as we're here, ditto for `.dig`?
```suggestion
          state: flow_session.dig(:pii_from_doc, :state),
```